libcloud/compute/drivers/ec2.py

```diff
@@ -2157,6 +2163,9 @@ def create_volume(self, size, name, location=None, snapshot=None,
             if ex_kms_key_id is not None:
                 params['KmsKeyId'] = ex_kms_key_id
 
+        if ex_volume_type == 'gp3' and ex_throughput:
```

Comment:
I believe we want to allow caller to specify throughput for io1 and io2 volume types, similar to iops.

Reply:
According to the [EC2 API reference](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVolume.html):

>  Throughput
> The throughput to provision for a volume, with a maximum of 1,000 MiB/s.
> This parameter is valid only for gp3 volumes. 

Are you seeing different behavior i.e. are you able to actually set the throughput for a volume type other than `gp3`? If so, the API reference might be outdated.

Regarding iops, my change already has the following logic that should allow you to set `Iops` for io1, io2 and gp3 volumes:
```
    if ex_volume_type in ['io1', 'io2', 'gp3'] and ex_iops:
            params['Iops'] = ex_iops
```

Reply:
You're right. We can't set throughput for io1 and io2 volumes, i.e. AWS Console volume modify doesn't have a throughput change option.